src/services/utilities.ts

```diff
@@ -2,7 +2,7 @@
 /* @internal */
 namespace ts {
     export const scanner: Scanner = createScanner(ScriptTarget.Latest, /*skipTrivia*/ true);
-    export const emptyArray: any[] = [];
+    export const emptyArray: never[] = [] as never[];

```

Comment:
Yes, otherwise I get `undefined[] is not assignable to never[]` error. May be related to #16651.